repository/CargoPackageManager-Calypso-Browser/PBSelectPackagesDialog.class.st

```diff
@@ -0,0 +1,207 @@
+Class {
+	#name : #PBSelectPackagesDialog,
```

Comment:
Notice that there is reusable search dialog in Calypso. For packages it would be:
```Smalltalk
   ^browser searchDialog requestSingleObject: 'Select package' from: ClyAllPackages sorted
```

Reply:
That's not useful for Cargo. I want to control what are the packages that are shown, that dialog is supposed to be reusable in different context and not only show system packages.